cmd/vic-machine/create/create.go

```diff
@@ -485,7 +485,7 @@ func (c *Create) Flags() []cli.Flag {
 
 		cli.BoolFlag{
 			Name:        "force, f",
-			Usage:       "Force the install, removing existing if present",
```

Comment:
Yeah, I just read this issue sorry I did not look at it ahead of time. I will leave the comment on karthiks issue. @karthik-narayan please take a look at this as well please 😄